plutus-ledger-api/src/PlutusLedgerApi/Common/Versions.hs

```diff
@@ -70,6 +71,10 @@ builtinsIntroducedIn = Map.fromList [
           ]),
   ((PlutusV2, ProtocolVersion 7 0), Set.fromList [
           SerialiseData, VerifyEcdsaSecp256k1Signature, VerifySchnorrSecp256k1Signature
+          ]),
+  -- FIXME: Protocol version TBD
```

Comment:
Hmm, maybe we should just move the constants from `ProtocolVersions.hs` to the main library and use those here?

Reply:
I thought so too, but then it is also nice to see the number of protocol version incrementing than just some names changing.